src/components/Employee.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { dateFormat } from '../utils/dateFormatter';
+
+export default class Employee extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { employee } = this.props;
+    return (
+      <>
+        {employee && (
```

Comment:
I would not set this condition right here. Why component what does not has any logic should know about any condition, it should be responsive just for rendering some content
```suggestion
{employee && <Employe employee={employee}/>}
```
I would do this before the call